src/WindowManager.vala

```diff
@@ -95,7 +99,12 @@ namespace Gala {
         private GLib.Settings animations_settings;
         private GLib.Settings behavior_settings;
 
+        private bool animating_switch_workspace = false;
+        private GestureAnimationDirector gesture_animation_director;
```

Comment:
Could you clear as to why is there a reason to have two `GestureAnimationDirector`'s declared in WindowManager and in MultitaskingView? Is it because we want to track handlers for the two classes separately?

Reply:
No reason at all, the same director can be used and passed in the constructor